.goreleaser.yml

```diff
@@ -12,8 +12,9 @@ brews:
   folder: HomebrewFormula
   install: bin.install "replicated"
 universal_binaries:
-- replace: true
-  id: replicated
+- ids:
+  - cli
+  replace: true
```

Comment:
I think it should be
```
universal_binaries:
  - replace: true
    id: cli 
    name_template: replicated
```
At least that looks correct local for me. @tdensmore Can you verify?

Reply:
Yes ID is deprecated in favor of IDS so this looks correct